sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceDeleteSuccessEventData.java

```diff
@@ -243,18 +276,44 @@ public ResourceDeleteSuccessEventData setAuthorization(String authorization) {
      * Get the claims property: The properties of the claims.
      *
      * @return the claims value.
+     * @deprecated This method is no longer supported since v4.9.0.
+     * <p> Use {@link ResourceDeleteSuccessEventData#getResourceClaims()} instead.
      */
+    @Deprecated
     public String getClaims() {
-        return this.claims;
+        return this.claimsString;
```

Comment:
This will always return null since the `setClaims` method is never called. The deserializer only sets the Map<String, String> value. The get method should check that the map is not empty and if it's not, serialize it to a string and then return it. Same thing for other string properties now.